distributed/hdfs.py

```diff
@@ -46,6 +49,8 @@ def read_bytes(fn, executor=None, hdfs=None, lazy=False, delimiter=None, **hdfs_
     blocks = get_block_locations(hdfs, fn)
     filenames = [d['filename'] for d in blocks]
     offsets = [d['offset'] for d in blocks]
+    if not_zero:
+        offsets = [max([o, 1]) for o in offsets]
```

Comment:
The header could be quite big, and in many cases we will have already read it, better to skip the read than try to deal with it later. Would be good to specify the initial offset, maybe.